tests/gnrc_ipv6_nib/main.c

```diff
@@ -449,45 +449,15 @@ static void test_handle_pkt__nbr_sol__invalid_dst(void)
     TEST_ASSERT_EQUAL_INT(0, msg_avail());
 }
 
-static void test_handle_pkt__nbr_sol__invalid_sl2ao(void)
-{
-    gnrc_ipv6_nib_nc_t nce;
-    void *state = NULL;
-    size_t icmpv6_len = _set_nbr_sol(&ipv6_addr_unspecified, &_loc_sol_nodes,
-                                     255U, 0U, &_loc_ll, _rem_l2,
-                                     sizeof(_rem_l2));
-
-    gnrc_ipv6_nib_handle_pkt(_mock_netif, ipv6, icmpv6, icmpv6_len);
-    TEST_ASSERT_MESSAGE(!gnrc_ipv6_nib_nc_iter(0, &state, &nce),
-                        "There is an unexpected neighbor cache entry");
-    /* TODO: check other views as well */
-    TEST_ASSERT_EQUAL_INT(0, msg_avail());
-}
-
-static void test_handle_pkt__nbr_sol__tgt_not_assigned(void)
-{
-    gnrc_ipv6_nib_nc_t nce;
-    void *state = NULL;
-    size_t icmpv6_len = _set_nbr_sol(&_rem_ll, &_loc_sol_nodes,
-                                     255U, 0U, &_rem_ll, _rem_l2,
-                                     sizeof(_rem_l2));
-
-    gnrc_ipv6_nib_handle_pkt(_mock_netif, ipv6, icmpv6, icmpv6_len);
-    TEST_ASSERT_MESSAGE(!gnrc_ipv6_nib_nc_iter(0, &state, &nce),
-                        "There is an unexpected neighbor cache entry");
-    /* TODO: check other views as well */
-    TEST_ASSERT_EQUAL_INT(0, msg_avail());
-}
-
```

Comment:
This are not removed, but rather moved down, since `test_handle_pkt__nbr_sol__invalid_sl2ao()` now also requires to check if a neighbor advertisement is in queue (so it requires the further down defined `test_pkt_is_nbr_adv()`.